### DISCUSSION TITLE
Timesheet optimization

### DISCUSSION THREAD

**Discussion by ftrueck** [Leantime/leantime#985] [Ideas]
The Timesheet in the project view could need some overhaul.

Current situation: 
- You select a project
- You click on Timesheet
- You see the hours for the selected time-period of this project
- You can sign the hours as approved / invoiced

There is nothing wrong with that. It works perfectly fine for one project. But there is the catch: If you have multiple projects you have to do this for every project each time you want to invoice or check or sth. else.

Optimized version:
- Global time sheet link
- You can select the project as an optional filter IN the timesheet filter section
- approved by mgr and invoiced checkboxes (quickactions) remain the same as the project version

This way it would be possible to manage time sheets for the projects all at once instead of switching around.

**Comment by gloriafolaron** [comment]
In case I don't remember to add it over there, I highly suggest moving this idea over to the issues board as a feature request.

Would love to get this added to our feature list.

Thanks for suggesting it!

**Reply by gloriafolaron** [reply]
@marcelfolaron

**Comment by marcelfolaron** [comment]
I finished this feature last night and it will go out with the next release.
